build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/testfixtures/TestFixturesPlugin.java

```diff
@@ -131,12 +131,20 @@ public void apply(Project project) {
             return composePath != null ? composePath : "/usr/bin/docker-compose";
         }));
 
-        tasks.named("composeUp").configure(t -> {
+        tasks.withType(ComposeUp.class).named("composeUp").configure(t -> {
             // Avoid running docker-compose tasks in parallel in CI due to some issues on certain Linux distributions
             if (BuildParams.isCi()) {
                 t.usesService(dockerComposeThrottle);
```

Comment:
Can we remove this throttle now that we are effectively throttling these things on the docker support service?

Reply:
I guess we can. will do